Automattic/wp-calypso
# Readymade Templates: Use live previews

Related to https://github.com/Automattic/dotcom-forge/issues/8138

## Proposed Changes

Readymade Templates now use a live preview on the collection screen rather than a hard-coded screenshot.

## Why are these changes being made?


This ensures that the preview is always accurate and doesn't have to be manually curated.


## Testing Instructions



 1. Apply D155716-code to your sandbox
 2. Sandbox public-api
 3. Use the calypso live link to go to /patterns
 4. Scroll down and look at the readymade template section
 5. Note that they display the contents from the stored RT rather than being hardcoded to neonfit
 6. Click on a RMT and notice that preview works in the details page

props to @mmtr for his original live preview work, which I've butchered here.

Before | After
-------|------
 <img width="2068" alt="Screenshot 2024-07-19 at 14 26 19" src="https://github.com/user-attachments/assets/d8f5b7ca-6228-4ad5-9cc9-0592dfe871e1">| <img width="2068" alt="Screenshot 2024-07-19 at 14 26 57" src="https://github.com/user-attachments/assets/058b7604-315b-41e6-8622-5e4f74803bb1">


## Pre-merge Checklist
